compressed_image_transport/src/compressed_subscriber.cpp

```diff
@@ -92,6 +92,14 @@ void CompressedSubscriber::internalCallback(const sensor_msgs::CompressedImageCo
   // Copy message header
   cv_ptr->header = message->header;
 
+  // Check for an empty input
+  if (message->data.empty())
+  {
+    ROS_WARN_STREAM_THROTTLE(30.0, "Image message was empty. This warning message is throttled every 30 seconds.");
```

Comment:
Isn't this going to be a pretty common occurrence? Should we have a message here at all?

Reply:
not sure, thats a good point as well. for us we dont need it, but maybe if someone else somehow triggers this by some bad setup it could be nice to have a warning.

Reply:
this could get triggered by any subscriber to a compressed topic. we just so happen to do that the most.

Reply:
I would just remove it to reduce the excessive logging. Its definitely also not a warning since this could happen in a normal use case.